core/services/ocrcommon/transmitter_pipeline.go

```diff
@@ -0,0 +1,84 @@
+package ocrcommon
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink/core/chains/evm/txmgr"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/pipeline"
+)
+
+const txObservationSource = `
+    transmit_tx [type=ethtx
```

Comment:
ah is this just to show the eth tx in the UI? I thought there was already an eth transactions page